### ISSUE TITLE
Tornado errors when inserting any object

### ISSUE THREAD

**Issue by ATranimal** [DDMAL/Neon.js#129]
Tornado is throwing post errors relating to pymei and XMLImport (related to error #105) every time an object is added to the editor. I'm assuming it's because pymei is still not working correctly, and the webserver is trying to edit the MEI file everytime something is changed.
```
ERROR:tornado.application:Uncaught exception POST /edit/squarenote/blank_test/insert/system (::1)

HTTPServerRequest(protocol='http', host='localhost:8080', method='POST', uri='/edit/squarenote/blank_test/insert/system', version='HTTP/1.1', remote_ip='::1', headers={'Origin': 'http://localhost:8080', 'Content-Length': '76', 'Accept-Language': 'en-US,en;q=0.8', 'Accept-Encoding': 'gzip, deflate, br', 'Host': 'localhost:8080', 'Accept': '*/*', 'User-Agent': 'Mozilla/5.0 (Macintosh; Intel Mac OS X 10_11_4) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/58.0.3029.96 Safari/537.36', 'Connection': 'keep-alive', 'X-Requested-With': 'XMLHttpRequest', 'Pragma': 'no-cache', 'Cache-Control': 'no-cache', 'Referer': 'http://localhost:8080/editor/blank_test.mei', 'Content-Type': 'application/x-www-form-urlencoded; charset=UTF-8', 'Cookie': 'sessionid=1m30s5z11g9ohv7py4vsfap6edt83mti; csrftoken=0YaLh778sOlX1bOpgi8R3CmEMRH9YcjN0TKxXWMENF4DP55sTk7WNzqVnOfgMgWi'})
Traceback (most recent call last):

  File "/Users/andrewtran/Documents/Neon.js/app_env/lib/python2.7/site-packages/tornado/web.py", line 1509, in _execute

    result = method(*self.path_args, **self.path_kwargs)

  File "/Users/andrewtran/Documents/Neon.js/neonsrv/tornadoapi.py", line 389, in post
    md = ModifyDocument(fname)

  File "/Users/andrewtran/Documents/Neon.js/neonsrv/modifymei.py", line 6, in __init__
    self.mei = XmlImport.read(filename)

  File "build/bdist.macosx-10.11-x86_64/egg/pymei/__init__.py", line 29, in read
    doc = XmlImport.documentFromFile(filename)

RuntimeError: The file /Users/andrewtran/Documents/Neon.js/MEI_DIRECTORY/squarenote/blank_test is malformed.

ERROR:tornado.access:500 POST /edit/squarenote/blank_test/insert/system (::1) 4.03ms
```

**Comment by ATranimal** [comment]
With commit 77954393f3f1d62f481124842a26d98e0af95104 we now have new issues in tornado when inserting objects

```
Traceback (most recent call last):

  File "/Users/andrewtran/Documents/Neon.js/app_env/lib/python2.7/site-packages/tornado/web.py", line 1509, in _execute
    result = method(*self.path_args, **self.path_kwargs)

  File "/Users/andrewtran/Documents/Neon.js/neonsrv/tornadoapi.py", line 31, in post
    result = md.insert_punctum(before_id, pname, oct, dot_form, ulx, uly, lrx, lry)

  File "/Users/andrewtran/Documents/Neon.js/neonsrv/modifymei.py", line 62, in insert_punctum
    parent = before.getParent()

AttributeError: 'NoneType' object has no attribute 'getParent'
```

**Comment by ATranimal** [comment]
Fixed also in 5d2e3091194fa19f970337fdff906d21050f8e20 with conjunction with #139
